api/src/main/java/com/okta/idx/sdk/api/client/WrapperUtil.java

```diff
@@ -33,6 +33,7 @@
     static AuthenticationResponse handleIllegalArgumentException(IllegalArgumentException e) {
         AuthenticationResponse authenticationResponse = new AuthenticationResponse();
         logger.error("Exception occurred", e);
+        authenticationResponse.addError(e.getMessage());
```

Comment:
Should this stay? Seems like it might have only been used for debugging.

Reply:
This should stay. This is the actual error from exception trace and would be useful to the developer.

Reply:
removed